experimental/testdata/frame-custom-meta.golden.txt

```diff
@@ -36,5 +36,5 @@ Dimensions: 1 Fields by 3 Rows
-FRAME=QVJST1cxAAD/////gAEAABAAAAAAAAoADgAMAAsABAAKAAAAFAAAAAAAAAEDAAoADAAAAAgABAAKAAAACAAAAFgAAAACAAAAKAAAAAQAAAAg////CAAAAAwAAAAAAAAAAAAAAAUAAAByZWZJZAAAAED///8IAAAAFAAAAAkAAABGcmFtZSBUd28AAAAEAAAAbmFtZQAAAAABAAAAGAAAAAAAEgAYABQAAAATAAwAAAAIAAQAEgAAABQAAACwAAAAtAAAAAAAAAWwAAAAAwAAAGgAAAA0AAAABAAAAKz///8IAAAAGAAAAA0AAABzaW5nbGUgc3RyaW5nAAAABAAAAG5hbWUAAAAA2P///wgAAAAUAAAACQAAAHsiYSI6ImIifQAAAAYAAABsYWJlbHMAAAgADAAIAAQACAAAAAgAAAAgAAAAFQAAAHsiZGlzcGxheU5hbWUiOiIxMjMifQAAAAYAAABjb25maWcAAAAAAAAEAAQABAAAAA0AAABzaW5nbGUgc3RyaW5nAAAAAAAAAP////+YAAAAFAAAAAAAAAAMABYAFAATAAwABAAMAAAAGAAAAAAAAAAUAAAAAAAAAwMACgAYAAwACAAEAAoAAAAUAAAASAAAAAMAAAAAAAAAAAAAAAMAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAQAAAAAAAAABAAAAAAAAAACAAAAAAAAAAAAAAAAQAAAAMAAAAAAAAAAAAAAAAAAAAAAAAAAQAAAAIAAAADAAAAYWJjAAAAAAAQAAAADAAUABIADAAIAAQADAAAABAAAAAsAAAAOAAAAAAAAwABAAAAkAEAAAAAAACgAAAAAAAAABgAAAAAAAAAAAAAAAAAAAAAAAoADAAAAAgABAAKAAAACAAAAFgAAAACAAAAKAAAAAQAAAAg////CAAAAAwAAAAAAAAAAAAAAAUAAAByZWZJZAAAAED///8IAAAAFAAAAAkAAABGcmFtZSBUd28AAAAEAAAAbmFtZQAAAAABAAAAGAAAAAAAEgAYABQAAAATAAwAAAAIAAQAEgAAABQAAACwAAAAtAAAAAAAAAWwAAAAAwAAAGgAAAA0AAAABAAAAKz///8IAAAAGAAAAA0AAABzaW5nbGUgc3RyaW5nAAAABAAAAG5hbWUAAAAA2P///wgAAAAUAAAACQAAAHsiYSI6ImIifQAAAAYAAABsYWJlbHMAAAgADAAIAAQACAAAAAgAAAAgAAAAFQAAAHsiZGlzcGxheU5hbWUiOiIxMjMifQAAAAYAAABjb25maWcAAAAAAAAEAAQABAAAAA0AAABzaW5nbGUgc3RyaW5nAAAAqAEAAEFSUk9XMQ==
+FRAME=QVJST1cxAAD/////kAEAABAAAAAAAAoADgAMAAsABAAKAAAAFAAAAAAAAAEEAAoADAAAAAgABAAKAAAACAAAAJwAAAADAAAAVAAAACgAAAAEAAAAFP///wgAAAAMAAAAAAAAAAAAAAAFAAAAcmVmSWQAAAA0////CAAAABQAAAAJAAAARnJhbWUgT25lAAAABAAAAG5hbWUAAAAAXP///wgAAAAsAAAAIAAAAHsiY3VzdG9tIjp7InNvbWVWYWx1ZSI6InZhbHVlIn19AAAAAAQAAABtZXRhAAAAAAEAAAAYAAAAAAASABgAFAAAABMADAAAAAgABAASAAAAFAAAAHgAAACAAAAAAAAAA4AAAAACAAAANAAAAAQAAADc////CAAAABAAAAAGAAAAbnVtYmVyAAAGAAAAdHN0eXBlAAAIAAwACAAEAAgAAAAIAAAAHAAAABIAAAB7InVuaXQiOiJQZXJjZW50In0AAAYAAABjb25maWcAAAAAAAAAAAYACAAGAAYAAAAAAAIADgAAAFNpbmdsZSBmbG9hdDY0AAD/////iAAAABQAAAAAAAAADAAWABQAEwAMAAQADAAAACgAAAAAAAAAFAAAAAAAAAMEAAoAGAAMAAgABAAKAAAAFAAAADgAAAAFAAAAAAAAAAAAAAACAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAKAAAAAAAAAAAAAAAAQAAAAUAAAAAAAAAAAAAAAAAAACF61G4HoUgQGZmZmZmZiFApHA9CtejLUCkcD0K1yMkQArXo3A9CiFAEAAAAAwAFAASAAwACAAEAAwAAAAQAAAALAAAADwAAAAAAAQAAQAAAKABAAAAAAAAkAAAAAAAAAAoAAAAAAAAAAAAAAAAAAAAAAAAAAAACgAMAAAACAAEAAoAAAAIAAAAnAAAAAMAAABUAAAAKAAAAAQAAAAU////CAAAAAwAAAAAAAAAAAAAAAUAAAByZWZJZAAAADT///8IAAAAFAAAAAkAAABGcmFtZSBPbmUAAAAEAAAAbmFtZQAAAABc////CAAAACwAAAAgAAAAeyJjdXN0b20iOnsic29tZVZhbHVlIjoidmFsdWUifX0AAAAABAAAAG1ldGEAAAAAAQAAABgAAAAAABIAGAAUAAAAEwAMAAAACAAEABIAAAAUAAAAeAAAAIAAAAAAAAADgAAAAAIAAAA0AAAABAAAANz///8IAAAAEAAAAAYAAABudW1iZXIAAAYAAAB0c3R5cGUAAAgADAAIAAQACAAAAAgAAAAcAAAAEgAAAHsidW5pdCI6IlBlcmNlbnQifQAABgAAAGNvbmZpZwAAAAAAAAAABgAIAAYABgAAAAAAAgAOAAAAU2luZ2xlIGZsb2F0NjQAAMABAABBUlJPVzE=
```

Comment:
These are here because of #580

This is not required since the match is still valid, but seems better to have this in sync with what would be produced today